### PR TITLE
Updated recommended Node version

### PR DESCRIPTION
**Pull request by timwright12**
## Description
Upgrading required node version to current LTS. https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/9837

After running a yarn or npm install, if you have a node version less than 14.15 it will let you know that you should install nmv and upgrade locally.

## Testing done
Running yarn install with node v.10.15.3 and 14.15.0

## Screenshots
Screen Shot 2020-11-04 at 4 22 05 PM

## Definition of done
- [ ] Changes have been tested in vets-website
- [X] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by U-DON** [comment]
I think the Dockerfile needs to be updated as well since it's used in the Jenkins pipeline.

**Comment by U-DON** [comment]
We should try to separate the disabled tests from this PR. If they're too unreliable to get the build passing, we can discuss with VAOS to resolve or disable them from their end.

File: src/applications/vaos/tests/new-appointment/components/DateTimeRequestPage.unit.spec.jsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { FETCH_STATUS } from '../../../utils/constants';
 import { waitFor } from '@testing-library/dom';
 import { Route } from 'react-router-dom';
 
-describe('VAOS <DateTimeRequestPage>', () => {
+xdescribe('VAOS <DateTimeRequestPage>', () => {
```
Comment (Clark-W-Griswold) [inline src/applications/vaos/tests/new-appointment/components/DateTimeRequestPage.unit.spec.jsx:12]: Is this change intended?

**Review by Clark-W-Griswold — APPROVED** [review]
👍

File: src/applications/vaos/tests/new-appointment/components/DateTimeRequestPage.unit.spec.jsx (hunk 1 above)
Comment (timwright12) [reply src/applications/vaos/tests/new-appointment/components/DateTimeRequestPage.unit.spec.jsx:12]: @Clark-W-Griswold yes, VAOS some tests were failing, so that was the recommendation from them to just skip the breaking tests

Comment (jbalboni) [reply src/applications/vaos/tests/new-appointment/components/DateTimeRequestPage.unit.spec.jsx:12]: I believe I have these fixed in master, so you could probably remove the `x` now.

Comment (timwright12) [reply src/applications/vaos/tests/new-appointment/components/DateTimeRequestPage.unit.spec.jsx]: @jbalboni I removed this instance, but another was still failing, so I left that one in

**Review by jbalboni — APPROVED** [review]

**Comment by ElijahLynn** [comment]
Need to also update https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/getting-started/ to reflect new version. cc @meganhkelley
